omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/ui/OmnipodOverviewFragment.kt

```diff
@@ -542,6 +542,19 @@ class OmnipodOverviewFragment : DaggerFragment() {
         }
     }
 
+    private fun readableZonedTimeRelative(time: DateTime): String {
```

Comment:
Not necessarily for this PR, just a remark: There is quite a lot of general format utility functions in a Fragment.
We could move them to DateUtil as well.